novawallet/Common/Services/ExtrinsicService/Substrate/ExtrinsicSenderResolution/Delegation/DelegationResolutionGraph.swift

```diff
@@ -0,0 +1,243 @@
+import Foundation
+import SubstrateSdk
+
+enum DelegationError: Error {
+    case noCompatibleProxyType
+    case invalidMultisigParameters
+    case noValidPath
+    case emptyPaths
+    case disjointPaths
+}
+
+struct DelegationKey: Hashable {
+    let delegate: AccountId
+    let delegated: AccountId
+}
+
+protocol AccountDelegationGraphValue {
```

Comment:
I would manually emphasize that it is protocol. For example, DelegationResolutionNodeProtocol